test/test_strerror.cpp

```diff
@@ -21,10 +21,11 @@
 #include <dirent.h>
 #endif
 
-#include <string>
-
+#include "rcutils/macros.h"
 #include "rcutils/strerror.h"
 
+#include "./mimick.h"
```

Comment:
@Blast545 nit: 
```suggestion
#include "mimick.h"
```

Reply:
I don't like it either, but `cpplint` complains if I don't include the folder path for `.h` files.

Reply:
I see. This is wrong, and it's because of `mimick` includes' layout. See https://github.com/ros2/Mimick/pull/7.